packages/material-ui/src/ButtonBase/TouchRipple.js

```diff
@@ -63,7 +63,7 @@ export const TouchRippleRoot = experimentalStyled(
 // in string templates. Do not convert these styles in JS object as it will break.
 export const TouchRippleRipple = experimentalStyled(
   Ripple,
-  {},
+  { shouldForwardProp: (prop) => shouldForwardProp(prop) || prop === 'classes' },
```

Comment:
This is the only place so far that we need to actually forward the `classes` prop, as it is expected in the `Ripple`